librz/debug/debug.c

```diff
@@ -1636,7 +1625,7 @@ RZ_API bool rz_debug_is_dead(RzDebug *dbg) {
 	if (!strcmp(dbg->cur->name, "io")) {
 		return false;
 	}
-	bool is_dead = (dbg->pid == -1 && strncmp(dbg->cur->name, "gdb", 3)) || (dbg->reason.type == RZ_DEBUG_REASON_DEAD);
+	bool is_dead = (dbg->pid == -1 && strncmp(dbg->cur->name, "gdb", 3) && strncmp(dbg->cur->name, "bochs", 5)) || (dbg->reason.type == RZ_DEBUG_REASON_DEAD);
```

Comment:
Could you please extract that `strncmp(gdb) && strncmp(bochs)` comparison into a separate inline function? And also, write a comment about why those in particular.